index.html

```diff
@@ -4,180 +4,145 @@
 permalink: index.html
 ---
 <div class="site-wrap">
-        <section class="container full col-xs-12">
-            <div class="info-block info-block-main">
-                <div class="row middle-xs">
-                    <div class="col-lg-7 col-md-7 col-xs-12 main-video">
-                        <div class="monero-video">
-                            <!--iframe width="560" height="315" src="https://www.youtube.com/embed/TZi9xx6aiuY" frameborder="0" allowfullscreen></iframe-->
-                            <video controls poster="/img/monero-community.png" preload="metadata">
-                                <source src={% if site.lang == 'en' %}"/media/Monero_Promo.m4v"{% else %}"/media/{{ site.lang }}/Monero_Promo.m4v"{% endif %}>
-                            </video>
-                        </div>
-                        
+    <section class="container full col-xs-12">
+        <div class="info-block info-block-main">
+            <div class="row middle-xs">
+                <div class="col-lg-7 col-md-7 col-xs-12 main-video">
+                    <div class="monero-video">
+                        <!--iframe width="560" height="315" src="https://www.youtube.com/embed/TZi9xx6aiuY" frameborder="0" allowfullscreen></iframe-->
```

Comment:
Is it worth just deleting this comment? We can always just copy it back out of the Git history.

Reply:
IIRC it was put there as a backup in case we had problems with the hosted video. I would just leae it there, since it doesn't really bother and we have a quick reference to the video on Youtube. I wouldn't rely on the git history, mostly because we would need to remember that once we had it and i think tha won't happen :)